src/dates/date.md

```diff
@@ -0,0 +1,209 @@
+As a beginner one of the most interesting topics you’ll learn are Dates. Even though it might sound a bit boring, it's one of the key things you might know; from your Database, API, GUI, and more learning how to deal with dates properly is a key factor in writing good applications. 
+
+# Before we start
+An important detail you must remember is Java was rushed for the web, thus you have a lot of quirky and somewhat stupid ways to do stuff. The goal of this guide will be to teach you how to use the latest Java APIs to write efficient good code. I’ll aim to teach you how not to code anything hard and rather use standardized ISO for a scalable application. 
+
+# How to work with Dates
+## The Date API 
+
+As of `Java 8 SE` with the release of the [Date Class](https://docs.oracle.com/javase/8/docs/api/java/util/Date.html) defined as:
```

Comment:
Sentences like these date the content. Keep in mind that we are writing assuming anonymous main methods are already in the language - which hasn't happened yet.

Someone learning Java 25 might be interested when stuff was added, but I don't feel the need to call attention to it. Also Java 8 wasn't when Date was added. That was Java 1.

Reply:
wasn't it released with jdk 1(more specifically jdk 1.1) with java 8? I'm tryna search online and find something that implies it was released with java 1 but I don't see any, also oracles java 1 core libraries list won't load up for some reason